src/components/header.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Image,
+  ListItem,
+  UnorderedList,
+  Text,
+} from "@chakra-ui/react";
+import { Link } from "react-router-dom";
+import { useAuth, Authenticated } from "../context/auth";
+
+const Header = () => {
+  const { signout } = useAuth();
+
+  return (
+    <Box paddingTop={5} paddingBottom={5}>
+      <Flex
+        justifyContent="space-between"
+        background="#7b8e93"
+        padding="10px 20px"
+        borderRadius="10px"
+        alignItems="center"
+      >
+        <Flex align="center">
+          <Link to={"/"} style={{ fontWeight: "bold" }}>
+            <Image
+              src="https://emojis.slackmojis.com/emojis/images/1534453840/4524/zlatan_ibrahimovic.png?1534453840"
+              display="inline-block"
+              height={10}
+            />
+            Zultan's Voting App
+          </Link>
+        </Flex>
+        <Flex alignItems="center">
+          <Text
+            display="inline-block"
+            fontWeight="bold"
+            mr="1rem"
+            lineHeight="1"
+          >
+            <Link to={"/"}>Home</Link>
+          </Text>
+          <Authenticated
```

Comment:
We don't need to get the `user` from the `useAuth` because it is done in the `Authenticated` wrapper. This is instead of `user ? <div>{user.name}</div> : <div>Must Login</div>` style code.